src/modules/commander/Commander.cpp

```diff
@@ -915,23 +915,20 @@ Commander::handle_command(const vehicle_command_s &cmd)
 
 	case vehicle_command_s::VEHICLE_CMD_DO_FLIGHTTERMINATION: {
 			if (cmd.param1 > 1.5f) {
+				// Test termination command triggers lockdown but not actual termination.
 				if (!_lockdown_triggered) {
 					_armed.lockdown = true;
 					_lockdown_triggered = true;
 					PX4_WARN("forcing lockdown (motors off)");
 				}
 
 			} else if (cmd.param1 > 0.5f) {
-				//XXX update state machine?
+				// Trigger real termination.
 				if (!_flight_termination_triggered) {
 					_armed.force_failsafe = true;
 					_flight_termination_triggered = true;
 					PX4_WARN("forcing failsafe (termination)");
-				}
-
-				if ((int)cmd.param2 <= 0) {
-					/* reset all commanded failure modes */
-					PX4_WARN("reset all non-flighttermination failsafe commands");
+					send_parachute_command();
 				}
 
 			} else {
```

Comment:
Interesting, the MAVLink specs don't tell you that you can un-terminate when sending a lower parameter 👀 
https://mavlink.io/en/messages/common.html#MAV_CMD_DO_FLIGHTTERMINATION

Reply:
Indeed. Something we could suggest to add to the spec.

Reply:
https://github.com/mavlink/mavlink/pull/1649